extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryHandlerValueFactory.java

```diff
@@ -11,14 +11,20 @@
 import io.sentry.Sentry;
 import io.sentry.SentryOptions;
 import io.sentry.config.Lookup;
+import io.sentry.dsn.Dsn;
 import io.sentry.jul.SentryHandler;
 
 @Recorder
 public class SentryHandlerValueFactory {
     private static final Logger LOG = Logger.getLogger(SentryConfigProvider.class);
 
     public RuntimeValue<Optional<Handler>> create(final SentryConfig config) {
+        final SentryConfigProvider provider = new SentryConfigProvider(config);
+        final Lookup lookup = new Lookup(provider, provider);
+
         if (!config.enable) {
+            // Disable Sentry
+            Sentry.init(SentryOptions.from(lookup, Dsn.DEFAULT_DSN));
```

Comment:
Looks a bit like dark magic to me :).

Reply:
It's a bit magic, but Sentry API is not really made for this kind of use case.

 I Already had to implement the `Lookup` in order to use our own config.

Just to make you understand this code, when there is no DSN, Sentry show a warning and set it to `Dsn.DEFAULT_DSN`, Using this just prevent from showing the warning..

Reply:
This is what I am referring to above: https://github.com/getsentry/sentry-java/blob/master/sentry/src/main/java/io/sentry/dsn/Dsn.java#L101